modules/electrophysiology_browser/ajax/get_electrophysiology_session_data.php

```diff
@@ -0,0 +1,336 @@
+<?php declare(strict_types=1);
+/**
+ * Get Electrophysiology Session Data.
+ *
+ * This retrieves the entire electrophysiology session data
+ * for a candidate to be visible on the electrophysiology_session form.
+ *
+ * PHP Version 7
+ *
+ * @category LORIS
+ * @package  Electrophysiology_Session
+ * @author   Muhammad Khan <muhammad.khan@mcin.ca>
+ * @license  LORIS license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+require_once 'ElectrophysioFile.class.inc';
+
```

Comment:
$_REQUEST['outputType'] shoudl also be checked early. Additionally, make sure the value is raw, derivatives or all_types. Otherwise, getFilesData will return all types even where an invalid type is provided.

Reply:
ok